.github/workflows/lint.yaml

```diff
@@ -14,6 +16,12 @@ jobs:
           python-version: 3.8
       - name: Install tox
         run: pip install tox
+      # yamllint is pretty good, but doesn't handle helm templates too well.
+      # Duplicate files and exclude lines with {{ in them.
```

Comment:
Should we completely exclude helm templates instead?
There's no guarantee at all that commenting the lines with `{{}}` will generate valid yaml no?

Reply:
I think we should try to include them. A lot of them are almost entirely yaml with a few replaces, or a marker at the beginning. Local may be the most "complex" but it really just has:
`{{ if .Values.localdev.enabled }}`
followed by ordinary yaml.

So yes, I think there is a lot of benefit, even if we have to nudge things a bit, to getting the templates reviewed as well. My views on this would likely change if we find that our helm templates increase in complexity and a bit of sed doesn't get us anything useful.